packages/babel-generator/src/generators/types.js

```diff
@@ -1,6 +1,8 @@
 import * as t from "@babel/types";
 import jsesc from "jsesc";
 
+const surrogates = /[\uD800-\uDBFF][\uDC00-\uDFFF]|([\uD800-\uDBFF]|[\uDC00-\uDFFF])/g;
```

Comment:
```suggestion
const surrogates = /([\uD800-\uDFFF])/gu;
```
given that it is ES2015 and should be supported by Node.js 6.

Reply:
Cool, done!